docs/src/main/java/io/quarkus/docs/generation/AssembleDownstreamDocumentation.java

```diff
@@ -61,6 +61,7 @@ public class AssembleDownstreamDocumentation {
             Pattern.CASE_INSENSITIVE + Pattern.MULTILINE);
     private static final String SOURCE_BLOCK_PREFIX = "[source";
     private static final String SOURCE_BLOCK_DELIMITER = "--";
+    private static final Pattern FOOTNOTE_PATTERN = Pattern.compile("footnote:([a-z0-9_-]+)\\[(\\])?");
```

Comment:
@gsmet Also, Guillaume, I think we need **"footnoteref:** instead of **"footnote:**. Am I right? We need to use that older macro, combined with the notion demonstrated here.